web/tests/buddy_data.test.js

```diff
@@ -226,6 +228,7 @@ test("title_data", ({override}) => {
         second_line: "translated: Not active in the last year",
         third_line: "",
         show_you: false,
+        is_deactivated: false,
     };
     assert.deepEqual(buddy_data.get_title_data(old_user.user_id, is_group), expected_data);
```

Comment:
I think it'd be worth adding an extra test case here for a deactivated user to cover the new code path; it seems like only a handful of lines of test code.

Reply:
I have added 2 more test cases. One for deactivated users and the other one for deactivated bots.